spring-data-relational/src/main/java/org/springframework/data/relational/core/dialect/AbstractDialect.java

```diff
@@ -132,6 +133,14 @@ public SelectRenderContext getSelectContext() {
 		}
 	}
 
+	/**
+	 * @return default implementation of InsertWithDefaultValues
+	 */
+	@Override
+	public InsertWithDefaultValues getSqlInsertWithDefaultValues() {
```

Comment:
This Default implementation should go in the interface, so it works also for those that don't use the `AbstractDialect`

Reply:
Resolved with [this](https://github.com/spring-projects/spring-data-jdbc/pull/1047/commits/71204add3936c7a8ee171d01b68ffe4f8a3d7b32)